main.py

```diff
@@ -0,0 +1,14 @@
+import sys
+from reader import Reader
+from parser import Parser
+
+if __name__ == '__main__':
+
+    if sys.argv[1] == '-e':
+        files_data = Reader().read_all_file('/home/faran/Downloads/weatherfiles (1)/weatherfiles')
+        calculation_instance = Parser().parser(files_data)
```

Comment:
separate out the calculations from parsing
`separation of concerns` and `loose couplin`